src/main/java/net/masterthought/cucumber/util/StepNameFormatter.java

```diff
@@ -31,6 +31,10 @@ public static String format(String stepName, Argument[] arguments, String preArg
 
     private static void surroundArguments(Argument[] arguments, String preArgument, String postArgument, String[] chars) {
         for (Argument argument : arguments) {
+            if (argument.getOffset() == null) {
+                continue;
+            }
+
             int start = argument.getOffset();
```

Comment:
move ```start``` above and use it for if condition

Reply:
also can we have this case included into sample.json file?

Reply:
As for moving `start` above, it is possible, but then `start` and `end` would be separated (contrary to what you asked below), because `end` depends on `start` which might be null.